package-lock.json

```diff
@@ -11105,7 +11105,7 @@
       "dependencies": {
         "source-map": {
           "version": "0.4.4",
-          "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.4.4.tgz",
+          "resolved": "http://registry.npmjs.org/source-map/-/source-map-0.4.4.tgz",
```

Comment:
what's happening here

Reply:
🤷‍♂ npm install with `npm@6.9.0` and `node@10.16.0` I think